.github/workflows/ci.yml

```
@@ -93,3 +93,70 @@ jobs:
       - name: Compile
         working-directory: build
         run: make -j4
+  windows-build:
+    runs-on: windows-2019
+    name: Windows CI
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    - name: Setup MSYS2
+      uses: msys2/setup-msys2@v2
+      with:
+        msystem: MINGW64
+        update: true
+        install: >-
+          git
+          make
+        pacboy: >-
+          toolchain:p
+          cmake:p
+          ninja:p
+          libusb:p
+          boost:p
+          qt5:p
+          spdlog:p
+          volk:p
+          libsndfile:p
+          soapysdr:p
+          fftw:p
+          rtl-sdr:p
+    - name: Clone and Build GNU Radio
+      shell: msys2 {0}
+      run: |
+        git clone --recursive https://github.com/cozycactus/gnuradio.git
+        cd gnuradio
+        git checkout fixmingw64
+        mkdir build && cd build
+        cmake -DCMAKE_INSTALL_PREFIX=C:/gqrx -G Ninja ..
+        ninja
+        ninja install
+    - name: Clone and Build gr-osmosdr
+      shell: msys2 {0}
+      run: |
+        git clone https://github.com/osmocom/gr-osmosdr.git
```

Better to use the official source instead of the GitHub mirror, as we do elsewhere in CI.
```suggestion
        git clone https://gitea.osmocom.org/sdr/gr-osmosdr.git
```